aws/internal/keyvaluetags/generators/servicetags/main.go

```diff
@@ -182,8 +187,10 @@ package keyvaluetags
 import (
 	"github.com/aws/aws-sdk-go/aws"
 {{- range .SliceServiceNames }}
+{{- if . | IncludeImport }}
```

Comment:
i had to add this annoying thing is that i had to add a "filter" for imports the the servicetags generation to skip wafregional import as the package itself is not used in this instance

Reply:
We can actually remove the custom function with [`eq`](https://golang.org/pkg/text/template/#hdr-Functions) 👍 

```go
{{- if eq . (. | TagPackage) }}
```